packages/ui-components/src/components/Chart/Table.js

```diff
@@ -89,22 +93,26 @@ const NoData = styled(SmallAlert)`
   margin-right: auto;
 `;
 
+const COLUMN_BLACKLIST = ['name', 'timestamp', 'dataElementCode'];
+
 const getColumns = rawData => {
   const columns = [];
   rawData.forEach(row => {
     Object.keys(row)
-      .filter(key => key !== 'name' && key !== 'timestamp')
+      .filter(key => !COLUMN_BLACKLIST.includes(key)) // Don't show columns that don't make sense
+      .filter(key => !columns.includes(key)) // de-dupe
+      .filter(key => typeof row[key] !== 'object') // filter out object values such as metadata
```

Comment:
Rather than blacklisting, let's follow the same logic as the charts themselves to get the columns, which is (from memory):
- Use the keys in `chartConfig`; and if `chartConfig` doesn't exist
- Use `value` as the only column